compiler/circle-mpqsolver/src/core/Dumper.cpp

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright (c) 2023 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "Dumper.h"
+
+#include <luci/CircleExporter.h>
+#include <luci/CircleFileExpContract.h>
+
+#include <json.h>
+#include <fstream>
+#include <sys/stat.h>
+
+using namespace mpqsolver::core;
+
+const std::string Dumper::default_dtype_key = "default_quantization_dtype";
+const std::string Dumper::default_granularity_key = "default_granularity";
+const std::string Dumper::layers_key = "layers";
+const std::string Dumper::model_key = "model_path";
+const std::string Dumper::layer_name_key = "name";
+const std::string Dumper::layer_dtype_key = "dtype";
+const std::string Dumper::layer_granularity_key = "granularity";
```

Comment:
Are these variables used outside of this file? If not, it would be better to define them in anonymous namespace.

Reply:
Ahhh. Ok.